webapp/src/services/index.ts

```diff
@@ -1,13 +1,40 @@
+// eslint-disable-next-line import/no-unresolved
```

Comment:
What's the need for this comment?

Reply:
The line below was failing on CI. Took this from mm-ui-lib.